neuraxle/steps/flow.py

```diff
@@ -0,0 +1,113 @@
+from neuraxle.base import MetaStepMixin, BaseStep, ExecutionContext, DataContainer, NonTransformableMixin, \
+    ExecutionMode, NonFittableMixin, ResumableStepMixin
+
+
+class ResumableMetaStepMixin(ResumableStepMixin):
+    def should_resume(self, data_container: DataContainer, context: ExecutionContext) -> bool:
```

Comment:
shouldn't this be defined already by default in the MetaStepMixin? Why would we want annother class if this chexks anyway for `if isinstance(self.wrapped, ResumableStepMixin):` ?

Reply:
I'm still not sure about this. I dont rlly think we need a new class. Something about this felt strange but I don't know how to put it in words. There is a problem 65% chances here according to my feelings.

Reply:
why are you re-comenting on outdated code ?